rts/pom.xml

```diff
@@ -90,28 +90,37 @@ Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
     </profile>
     <profile>
 	  <id>jdk11</id>
-          <activation>
```

Comment:
Remark: we normally only include changes related to the issue. Also notice that even if your goal was to clean up the white space there is a tab on the previous line (line 92).

I don't block PRs for spurious white space clean up because it is very difficult to stop the IDE from correcting white space (and I often make the same error myself and would not like my PRs being blocked because of it).

Reply:
Thanks for spotting that tab; I'll correct it and push another commit.

Thanks for clarifying this. So, should I open a new issue/create a new PR if I think a code would benefit from a bit of cleaning up?

Reply:
We don't clean up historical code because it makes analysing diffs problematic.
We have a policy somewhere about it so I will dig that out and pass it on.

Reply:
But yes, one PR per issue (unless multiple issues are tightly interrelated).

Reply:
Thanks for clarifying this :+1: